creator-node/src/services/prometheusMonitoring/prometheusRegistry.ts

```diff
@@ -176,4 +176,4 @@ export class PrometheusRegistry {
   }
 }
 
-module.exports = PrometheusRegistry
+module.exports = new PrometheusRegistry()
```

Comment:
had a circular dependency between monitoringQueue and serviceRegistry so had to make prometheusRegistry a singleton (which it should probably be anyway)